src/connection/class-orders.php

```diff
@@ -63,6 +63,22 @@ public static function get_connection_config( $args = [] ) {
 	 * @return array
 	 */
 	public static function get_connection_args() {
-		return array();
+		return array_merge(
+			self::get_shared_connection_args(),
+			array(
+				'statuses'   => array(
+					'type'        => array( 'list_of' => 'String' ),
```

Comment:
Statuses should probably be an Enum, eh?

Reply:
Yea, probably